File: system_template/test/1_no_namespace/data/system_template/python_script/_map.py
[
    # Json files
    {
        "source": "python_script_no_output.behavior.json",
        "target": AUTO_FLAT,
        "python_script": True
    },
    {
        "source": "python_script.behavior.json",
        "target": AUTO_FLAT,
        "python_script": True,
        "json_template": True,
        "scope": {
            "variable_from_scope": 1234
        }
    },
    {
        "source": "python_script.mcfunction",
        "target": AUTO_FLAT,
        "python_script": True,
        "scope": {
            "variable_from_scope": 1234
        }
    },
]

File: modular_mc/test/packs/data/modular_mc/ordered_entries/ordered_entries_124/_map.ts
export const MAP = [
	{
		source: "ordered_entry.behavior.json",
		target: ":autoFlat",
		jsonTemplate: true,
		onConflict: "merge",
		executionOrder: 2,
		scope: {
			input: "ENTRY 2",
		},
	},
	{
		source: "ordered_entry.behavior.json",
		target: ":autoFlat",
		jsonTemplate: true,
		onConflict: "merge",
		executionOrder: 1,
		scope: {
			input: "ENTRY 1",
		},
	},
	{
		source: "ordered_entry.behavior.json",
		target: ":autoFlat",
		jsonTemplate: true,
		onConflict: "merge",
		executionOrder: 4,
		scope: {
			input: "ENTRY 4",
		},
	},
	// ENTRY 3 is sent from another module
];
